README.md

```diff
@@ -111,12 +112,31 @@ location @extensionless-php {
 }
 ```
 
+If you've put your nginx configuration in a file in `sites_available`, link it and test nginx
+
+```sh
+function link_nginx(){
+  domain="$1"
+  sudo ln -s /etc/nginx/sites-available/$domain /etc/nginx/sites-enabled/$domain
+}
+link_nginx your_domain.com
+sudo nginx -t
```

Comment:
that being said, i dont think this part is really necessary, as the purpose of the intructions aren't really to instruct on how to configure nginx. 

Maybe changing the "An nginx configuration similar to the one below... " line to be more specific about the nginx config file and how/where it should be created, so that the user can adjust as needed